modules/videoio/test/test_ffmpeg.cpp

```diff
@@ -118,6 +118,9 @@ class CV_FFmpegWriteBigVideoTest : public cvtest::BaseTest
                     frame_s = Size(352, 288);
                 else if( tag == VideoWriter::fourcc('H', '2', '6', '3') )
                     frame_s = Size(704, 576);
+                else if( tag == VideoWriter::fourcc('H', '2', '6', '4') )
+                    // OpenH264 1.5.0 has resolution limitations, so lets use DCI 4K resolution
```

Comment:
@alalek Out of curiosity: what kind of resolution limitations? A maximum in resolution or only specific width/height combinations?